[MTransit & MWAN] Configure tunnel health checks (#7922)
cloudflare/cloudflare-docs: content/magic-wan/how-to/_index.md

# How to

> Review the tasks below for more information on running different taks for your Magic WAN configuration.

Reviewer: ```suggestion
You can run the following tasks for your Magic WAN configuration:
```
